README.md

```diff
@@ -28,6 +28,23 @@ Where `PORT` is any free port on your machine (typically it is 8888) and `PATH_T
 (typically it is something like `${HOME}/aiidalab`).
 The last line of the output of the command above will contain the link to access AiiDAlab in your browser.
 
+# Update requirements.txt
+
+First make sure you have python 3.7 available in your system.
+If that is the case, then adjust the [`Pipfile`](Pipfile) according to the latest releases.
+Then do:
+```
+pip install pipenv # If it is already installed, make sure it is the latest version.
+pipenv lock --python 3.7 # To generate Pipfile.lock (takes 1-2 minutes).
+pipenv lock --requirements > requirements.txt_stable # To extract the dependencies of the stable packages.
+pipenv lock --dev --requirements > requirements.txt_dev # To extract the dependencies of the development packages.
+cat requirements.txt_stable requirements.txt_dev | sort | uniq > requirements.txt
```

Comment:
I thought about this, and I actually prefer to keep it. I can modify the text saying something like "if you want to include the `[dev]` requirements, follow this path, otherwise just do `pipenv lock --requirements > requirements.txt`". 

The point is: if next time I will have a package with weird dependencies (say, aiida is set to be lower than 1.5.0, even though it is actually working) I don't want to search again for the information on how to handle it.

Reply:
Yes, would much prefer to keep those as "additional instructions", but not use or advocate this approach as the "standard way".

Reply:
I changed my mind. Soon pip won't allow us to create such inconsistent environments, therefore it makes total sense to be more strict here.

If a package wouldn't fit the environment, we simply won't include it.